modules/javafx.controls/src/main/java/javafx/scene/control/Skin.java

```diff
@@ -28,9 +28,22 @@
 import javafx.scene.Node;
 
 /**
- * Base class for defining the visual representation of user interface controls
+ * An interface for defining the visual representation of user interface controls
  * by defining a scene graph of nodes to represent the skin.
  * A user interface control is abstracted behind the {@link Skinnable} interface.
```

Comment:
shouldn't that be _to represent the skinnable_?

Reply:
I think this description is good.  How would you change it?

Reply:
removed the part after ```user interface controls```.